src/python/grpcio/grpc/_cython/_cygrpc/credentials.pyx.pxi

```diff
@@ -126,28 +126,69 @@ cdef class SSLSessionCacheLRU:
     grpc_shutdown()
 
 
+cdef int verify_peer_callback_wrapper(const char* servername, const char* cert, void* userdata) with gil:
+  if userdata == NULL:
+    print("Error! Callback function wasn't set!")
+    return 1
+  fn = <object>userdata
+  py_servername = None
+  if servername != NULL:
+    py_servername = <object>servername
+  py_cert = None
+  if cert != NULL:
+    py_cert = <object>cert
+  try:
+    result = fn(py_servername, py_cert)
+    if result:
+      return 0
+    return 1
+  except Exception:
+    return 1
+
+
+cdef void verify_peer_callback_destruct(void *userdata) with gil:
+  fn = <object>userdata
```

Comment:
Local field doesn't justify its existence - just use `<object>user_data` on the next line.